client/src/components/Doctor/DoctorInfo.tsx

```diff
@@ -1,28 +1,90 @@
 import { NavLink } from 'react-router-dom';
 import Avatar from '@mui/material/Avatar';
 import { Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import Badge from '@mui/material/Badge';
 import IUser from '../../Interfaces/post/IUser';
 
-const UserPostInfo = ({ user }: { user: IUser | null | undefined }) => (
-  <NavLink to={`/users/${user?.id}`} className="user-info-post">
-    <Box display="flex">
-      <Avatar
-        sx={{ width: '60px', height: '60px' }}
-        alt={user?.name}
-        src={user?.avatar}
-      />
-      <Box
-        display="flex"
-        alignItems="flex-start"
-        flexDirection="column"
-        justifyContent="space-around"
-      >
-        <h3 style={{ paddingLeft: '10px' }}>{user?.name}</h3>
-        <span style={{ paddingLeft: '10px', color: '#2D9B9B' }}>
-          {user?.DoctorInfo?.workplace}
-        </span>
+const StyledBadge = styled(Badge)(({ theme }) => ({
+  '& .MuiBadge-badge': {
+    backgroundColor: '#44b700',
+    color: '#44b700',
+    boxShadow: `0 0 0 2px ${theme.palette.background.paper}`,
+    '&::after': {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '100%',
+      borderRadius: '50%',
+      animation: 'ripple 1.2s infinite ease-in-out',
+      border: '1px solid currentColor',
+      content: '""',
+    },
+  },
+  '@keyframes ripple': {
+    '0%': {
+      transform: 'scale(.8)',
+      opacity: 1,
+    },
+    '100%': {
+      transform: 'scale(2.4)',
+      opacity: 0,
+    },
+  },
```

Comment:
WHY SO MUCH STYLING, ripple is native in mui and if you wanna disable it, use disableRipple prop

Reply:
this is from mui